h2o-py/tests/testdir_sklearn/pyunit_sklearn_regression_all_estimators.py

```diff
@@ -0,0 +1,170 @@
+        scores[estimator_cls].update(with_h2o_frames=score)
+
+
+def test_estimator_with_numpy_arrays(estimator_cls):
+    _ensure_connection_state(connected=False)
+    estimator = estimator_cls(init_connection_args=init_connection_args, **_get_default_args(estimator_cls))
+
+    data = _get_data(format='numpy')
+    assert isinstance(data.X_train, np.ndarray)
+
+    with estimator:
+        estimator.fit(data.X_train, data.y_train)
+        preds = estimator.predict(data.X_test)
+        print(preds)
+        assert isinstance(preds, np.ndarray)
+        if _get_custom_behaviour(estimator_cls).get('preds_as_vector', True):
+            assert preds.shape == (len(data.X_test),), "got {}".format(preds.shape)
+        else:
+            assert preds.shape[0] == len(data.X_test)
+
+        try:
+            estimator.predict_proba(data.X_test)
+        except AttributeError as e:
+            assert "No `predict_proba` method" in str(e)
+
+        if _get_custom_behaviour(estimator_cls).get('score', True):
+            score = estimator.score(data.X_test, data.y_test)
+            assert isinstance(score, float)
+            skl_score = r2_score(data.y_test, preds)
+            assert abs(score - skl_score) < 1e-6
+            scores[estimator_cls].update(with_numpy_arrays=score)
+
+
+def test_scores_are_equivalent(estimator_cls):
+    try:
+        lk, rk = ('with_h2o_frames', 'with_numpy_arrays')
+        est_scores = scores[estimator_cls]
+        if lk in est_scores and rk in est_scores:
+            assert abs(est_scores[lk] - abs(est_scores[rk])) < 1e-6, \
+                "expected equivalent scores but got {lk}={lscore} and {rk}={rscore}" \
+                    .format(lk=lk, rk=rk, lscore=est_scores[lk], rscore=est_scores[rk])
+        elif lk not in est_scores:
+            print("no scores for {}".format(estimator_cls.__name__+' '+lk))
+        else:
+            print("no scores for {}".format(estimator_cls.__name__+' '+rk))
+    except AssertionError as e:
+        if _get_custom_behaviour(estimator_cls).get('scores_may_differ', False):
+            print("ERROR !!! "+str(e))
+        else:
+            raise e
+
+
+def make_test(test, classifier):
+    bound_test = partial(test, classifier)
+    bound_test.__name__ = test.__name__
+    pyunit_utils.tag_test(bound_test, classifier.__name__)
+    return bound_test
+
+
+def make_tests(classifier):
+    return map(lambda test: make_test(test, classifier), [
+        test_estimator_with_h2o_frames,
+        test_estimator_with_numpy_arrays,
+        test_scores_are_equivalent
+    ])
+
+
+failing = [
+    'H2OCoxProportionalHazardsRegressor',  # doesn't support regression?
+    'H2ODeepWaterRegressor',  # requires DW backend
+    'H2OStackedEnsembleRegressor',  # needs a separate test (requires models as parameters),
+    'H2OSupportVectorMachineRegressor',  # doc says that our SVM estimator also supports regression, but it seems to require categorical target
+]
```

Comment:
reviewers, please confirm if this is normal...

Reply:
SVM only supports binary classification

cc: @angela0xdata

Reply:
@michalkurka http://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/svm.html#svm-algorithm
>>  This implementation can be used to solve binary classification problems.

I'll make it more prominent, though.

Reply:
thanks @angela0xdata 

@seb-h2o where did you see in doc SVM supports only regression?

Reply:
http://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/svm.html#introduction
```
support vector machines are supervised learning models with associated learning algorithms that analyze data used for classification and regression analysis
```
that's why I thought we were also supporting SVR

Reply:
I removed the SVM regressor.